index.css

```diff
@@ -82,22 +82,32 @@
 }
 .btn-primary{
   margin: 0.5rem 0.5rem;
-  height: 3rem;
-  width: 6rem;
+  height: 4rem;
+  width: 10rem;
+  font-size: 1.5rem;
 }
 .link{
   text-decoration: none;
   margin: auto;
 }
 .footer{
+  display: flex;
+  justify-content: center;
   background-color: #ffffff;
   position: fixed;
   bottom: 0rem;
   left: 0rem;
-  padding: 1rem auto;
+  padding: 1rem;
   width: 100%;
   font-family: 'Roboto Condensed', sans-serif;
```

Comment:
Primary  `font-family`  should be declared in `body`'s CSS to reduce our efforts of writing `font-family` for each element.